### ISSUE TITLE
[feat] Implement Mqtt standard specification

### ISSUE THREAD

**Issue by slobberbone** [zwave-js/zwavejs2mqtt#452]
It would be interesting to implement in this project the specification describe here :
https://homieiot.github.io/

However, MQTT doesn’t define the structure and content of these messages and their relation. An IoT device publishes data and provides interaction possibilities but a controlling entity will need to be specifically configured to be able to interface with the device.

The Homie convention defines a standardized way of how IoT devices and services announce themselves and their data on the MQTT broker.

It is thereby a crucial aspect on top of the MQTT protocol for automatic discovery, configuration and usage of devices and services.

Some other xxx2mqtt already purpose this and I think specification for MQTT is the futur of home automation.
It would be not changing the way of this project work actually, but add another way to publish.

I'm from the open source project NextDom (home automation software) and we are working on another solution from scratch. We want to organize NextDom directly with MQTT and Node-Red (Nextdom becoming a kind of orchestrator. In this way, this standard for MQTT will be very interesting for us of course, but also for all others home automation software.

Thanks for reading and give me some feed back about that !

**Comment by robertsLando** [comment]
I received a similar request on the onl zwave2mqtt too. I'm open to implement this as long as it doesn't introduce any breaking change or complication. If you would likke to help me in this we can speak about it

**Comment by slobberbone** [comment]
Yes of course, we are several people at NextDom, and this kind of request could interest other projects like HA ... we would be able to find some arms to work on it !

**Comment by robertsLando** [comment]
@slobberbone Awesome! :) LMK

**Comment by syepes** [comment]
Any updates on this?
It would be great to get this type of message publication to facilitate the device integration

**Comment by jacoscaz** [comment]
Hi all! Is anyone following this issue? Is there anyone interested in having Z-Wave JS UI adhere to the homie specification? @robertsLando have you ever been asked about it between the previous post (Jan 2022) and now (Jun 2024)?

**Comment by robertsLando** [comment]
No one is working on this AFAIK. I accept PR. Also this should be opt-in in case and not the default behaviour
